AncientReligions/events/on_action_events.txt

```diff
@@ -20,6 +20,530 @@ character_event = {
 	desc = "B"
```

Comment:
The first idea is what I'm attempting in my "hotfix" branch (it is so much more than a hotfix at this point), but I haven't verified that it works.

I like the second idea better, and think we should go with that.